### ISSUE TITLE
ROCm 3.7 rocm-clang-ocl package still points 3.5

### ISSUE THREAD

**Issue by ye-luo** [RadeonOpenCompute/ROCm#1192]
Noticed after today's update.

**Comment by adilad** [comment]
What exactly are you pointing to? Can you point me to the URL where it is referring to 3.5 in the 3.7 release?

**Comment by ye-luo** [comment]
On ubuntu 20.04
![Screenshot from 2020-08-20 20-23-47](https://user-images.githubusercontent.com/1454251/90841787-3291d500-e323-11ea-863e-4d71a48dd8a4.png)

**Comment by srinivamd** [comment]
>> Every upgrade of rocm is a deep pain.
Yes, the rocminstall.py script from https://github.com/srinivamd/rocminstaller makes an "upgrade" simpler. One used it to install the new revision. The only issue still is updating the kernel packages which would require manual steps - remove rock-dkms* packages, reboot kernel, then run rocminstall.py script to install (it will install kernel packages only if none are installed).
